jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -162,7 +162,7 @@ def mount_repo(
         self._mounted[name] = mounted_branch
 
     def unmount_repo(self, name: str):
-        if not self._mounted[name]:
+        if not name in self._mounted:
```

Comment:
JIC this line isn't equivalent, can we change it to `if name not in self._mounted`